### PR TITLE
t/479: Created the "Editor placeholder" guide

### LINKED ISSUE #479: Implement configurable editor's placeholder
We could support:

* [ ] `config.placeholder = 'Type here...'`,
* [ ] and even reading the `placeholder` from the editor element if the editor was initialised on a textarea.

We should also:
* [ ] Find a decent name for the utility https://github.com/ckeditor/ckeditor5-engine/issues/899
* [ ] Address the a11y contrast issue https://github.com/ckeditor/ckeditor5-theme-lark/issues/213
* [ ] Update documentation with the information about placeholders
    * [ ] A placeholder feature guide with a couple of editors?

Of course, it will depend on editor creator how to set the placeholder (e.g. some creators may have multiple editables and then they will have to support setting multiple placeholders in `config.placeholder`).

### PR DESCRIPTION
**Pull request by oleq**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Docs: Created the "Editor placeholder" guide. Closes #479.

---

### Additional information

Part of the https://github.com/ckeditor/ckeditor5/issues/479 constellation.

### CODE REVIEW

File: docs/features/placeholder.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Configuring the placeholder
+
+There are two different ways of configuring the editor placeholder text:
+
+### Using the DOM attribute
+
+Set the `placeholder` attribute on an element passed to the `Editor.create()` method (for instance {@link module:editor-classic/classiceditor~ClassicEditor.create `ClassicEditor.create()`}) to configure the placeholder:
+
+```html
+<div id="editor" placeholder="Type the content here!">
```
Comment (Reinmar) [inline docs/features/placeholder.md]: This does not look like a valid HTML. Could you verify this?

File: docs/features/placeholder.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+		console.error( error );
+	} );
+```
+
+## Styling the placeholder
+
+The editor placeholder text is displayed using a CSS pseudo–element (`::before`) related to the first empty element in the editor content (usually a paragraph). You can use the following snippet to change the appearance of the placeholder:
+
+```css
+.ck.ck-editor__editable > .ck-placeholder::before {
```
Comment (Reinmar) [inline docs/features/placeholder.md]: What in case of a placeholder added directly to the editable?

File: docs/features/placeholder.md (hunk 1 above)
Comment (oleq) [reply docs/features/placeholder.md]: image
You're right, we should use `data-placeholder` instead.

File: docs/features/placeholder.md (hunk 2 above)
Comment (oleq) [reply docs/features/placeholder.md]: We don't support editors created out of non-breakable content elements yet. I figured documenting it makes little sense considering we don't have an ETA.

File: docs/features/placeholder.md (hunk 1 above)
Comment (oleq) [reply docs/features/placeholder.md]: https://github.com/ckeditor/ckeditor5/issues/1517

File: docs/features/placeholder.md (hunk 2 above)
Comment (Reinmar) [reply docs/features/placeholder.md]: OK. I left a comment to not forget that in https://github.com/ckeditor/ckeditor5/issues/762.

**Pull request merged**